### DISCUSSION TITLE
[5.x]: Simple inventory

### DISCUSSION THREAD

**Discussion by wmdhosting** [craftcms/commerce#3520] [Ideas]
Having native multi locations inventory support is fantastic, but it seems to be designed primarily for large-scale players.

Is there any possibility to introduce a setting in Commerce for a SIMPLE inventory system? Ideally, this would involve tracking a single column without limiting the number of locations to just five.

![2024-05-23_12-12-15](https://github.com/craftcms/commerce/assets/76176291/0c7a2269-3ca7-468f-8b9d-db14436de8be)

So we dont need to think to find other solution to track inventory when there is native one.

**Comment by linear** [comment]
<p><a href="https://linear.app/craftcms/issue/PT-1756/[5x]-simple-inventory">PT-1756 [5.x]: Simple inventory</a></p>

**Comment by sunscreem** [comment]
I think this would be a great enhancement. I've just handed over a site upgraded from Commerce 4 > 5 and the client is struggling a little with these changes. 

I'd love to see a simplified version.

**Comment by BrandKits** [comment]
This would be very useful. The 5 locations limit is super limiting, I even have a client with more than 5 locations within a single art gallery… they need to track artworks within the space. 

Perhaps a SKUs limit could be a compromise?

**Comment by white-franks** [comment]
We also have a few clients for whom this would be great. The only have 1 warehouse and only want a stock field like in Craft 4.

Maybe it's an option to have a setting to use the inventory setup, or the "old" stock field for a variant?

**Comment by lukeholder** [comment]
Commerce 5.7 is now out with an improvement related to this. There is now a view option to hide and show inventory columns.

**Comment by lukeholder** [comment]
Also see this response: https://github.com/craftcms/commerce/discussions/3429#discussioncomment-17726624
